pkg/runner/step_test.go

```diff
@@ -228,49 +228,49 @@ func TestIsStepEnabled(t *testing.T) {
 
 	// success()
 	step := createTestStep(t, "if: success()")
-	assertObject.True(isStepEnabled(context.Background(), step))
+	assertObject.True(isStepEnabled(context.Background(), step.getStepModel().If.Value, step))
```

Comment:
Could probably be simplified to:
```suggestion
	assertObject.True(isStepEnabled(context.Background(), step.getIfExpression(), step))
```